crates/egui/src/widgets/image.rs

```diff
@@ -40,6 +40,8 @@ use crate::{
 ///     .paint_at(ui, rect);
 /// # });
 /// ```
+///
+const RENDER_TIME: Duration = Duration::from_millis(6);
```

Comment:
I created it because of
`This is likely to lead to us skipping frames.

If the frames are 20ms each, and we're at time=6ms, then we should schedule a repaint after 14ms.`

I thought adding a magic number in the code that might change in the future wouldn't make that much sense